blog/smart-testing-0.0.5.textile

```diff
@@ -0,0 +1,53 @@
+---
+date: 2017-11-28
+author: matousjobanek
+---
+
+h3. Highlights of this release
+
+p. In this release we didn't ship any new cool feature, it mainly contains improvements on the level of Smart Testing API.

```

Comment:
> In this release we didn't ship any new cool feature,

I wouldn't start like that... it somehow sounds negative. Just say it has mainly API improvements making integration with 3rd party tools easier.